server/server.js

```diff
@@ -11,17 +12,34 @@ const app = express();
 app.use(bodyParser.urlencoded({ extended: true }));
 app.use(cors());
 app.use(bodyParser.json());
-app.use(morgan('tiny'));
+app.use(morgan('dev'));
 app.use(express.static(path.resolve('client', 'dist')));
 
-app.get('/game_carousel_info', (req, res) => {
-  console.log('SERVICE HEADERS', req.headers);
-  const queryId = req.query ? req.query.id : 1;
-  db.getInfo(queryId, (err, data) => {
+const getReviewsInfo = (gameId) => {
```

Comment:
I see you added a reviews route here, nice! There's also a route I created specifically for you so you don't have to get so much data back haha. The route is /review_counts and should get you just the info you need so you don't have to parse through so much data/get so much unnecessary data sent back. You can use either though - up to you!